src/fablabot/cogs/channel_management.py

```diff
@@ -1,121 +1,269 @@
 
     @app_commands.command(name="create", description="Crée un nouveau salon dans la catégorie spécifiée.")
     @app_commands.describe(
         channel="Le nom du salon à créer",
         category="La catégorie dans laquelle créer le salon",
     )
+    async def create(self, interaction: Interaction, channel: str, category: CategoryChannel) -> None:
+        """Create a text channel in the passed category.
+
+        Args:
+            interaction (Interaction): The interaction object.
+            channel (str): The name of the channel to create.
+            category (CategoryChannel): The category to create the channel in.
+        """
+        log_request("text.create", interaction, channel=channel, category=category.name)
+        assert isinstance(interaction.user, Member)
+        if not category.permissions_for(interaction.user).manage_channels:
+            logger.warning("Insufficient permissions for manage_channels: %r", interaction.user)
+            await interaction.response.send_message(
+                "Vous n'avez pas la permission de créer des salons dans cette catégorie.", ephemeral=True
+            )
+            return
+        existing_names = {c.name for c in category.channels}
+        if channel in existing_names:
+            logger.info("Channel %r already exists in %r", channel, category.name)
+            await interaction.response.send_message(f"Un salon {channel!r} existe déjà dans {category.name}.", ephemeral=True)
+            return
+        new_channel = await category.create_text_channel(channel)
+        logger.info("Created text channel %r in category %r", new_channel, category.name)
+        await interaction.response.send_message(f"Le salon {new_channel.mention} a été créé dans {category.name}.")
+
+    @app_commands.command(name="rename", description="Renomme un salon textuel.")
+    @app_commands.describe(channel="Salon à renommer", new_name="Nouveau nom du salon")
+    async def rename(self, interaction: Interaction, channel: TextChannel, new_name: str) -> None:
+        """Rename a text channel.
+
+        Args:
+            interaction (Interaction): The interaction object.
+            channel (TextChannel): The channel to rename.
+            new_name (str): The new name of the channel.
+        """
+        log_request("text.rename", interaction, channel=channel.name, new_name=new_name)
+        assert isinstance(interaction.user, Member)
+        if not channel.permissions_for(interaction.user).manage_channels:
+            logger.warning("Insufficient permissions for rename: %r", interaction.user)
+            await interaction.response.send_message(
+                f"Vous n'avez pas la permission de renommer le salon {channel.mention}.", ephemeral=True
+            )
+            return
+        old_name = channel.name
+        await channel.edit(name=new_name)
+        logger.info("Renamed channel %r from %r to %r", channel, old_name, new_name)
```

Comment:
Update % syntax in favor of f-string
```suggestion
        logger.info(f"Renamed channel {channel!r} from {old_name!r} to {new_name!r")
```